First Pass Rough Copyediting Updates (June 2023) (#1368)
TRaSH-Guides/Guides: includes/hardlinks/folder-structure.md

!!! warning

    It doesn't really matter which path you use for your media and appdata.

    The only thing you should avoid is `/home` (because folders in `/home` have restrictive permissions by default).

    As such, using `/home` could end up creating a permissions mess, so it's better to avoid this situation.

For this example, we're going to make use of a share called `data`.

The `data` folder has subfolders for `torrents` and `usenet`, and each of these have subfolders for `tv`, `movie`, `books` and `music` downloads to keep things neat. The `media` folder has nicely named `TV`, `Movies`, `Books` and `Music` subfolders. *This is your library and what you’d pass to Plex, Emby or JellyFin.*

> **Note:** In these examples, I'm purposefully using lowercase on all folders, as Linux is case-sensitive.

Reviewer: Why not use `!!! note` ?

    It doesn't really matter which path you use for your media and appdata,

>     the only thing you should avoid is `/home` (because folders in `/home` are expected to have some restrictive permissions).

Reviewer 2: ```suggestion
    the only thing you should avoid is `/home` (because folders in `/home`  do have restrictive permissions by default).
```

Author: Updated this one
